code/modules/mob/living/carbon/human/dummy.dm

```diff
@@ -103,6 +103,33 @@ INITIALIZE_IMMEDIATE(/mob/living/carbon/human/dummy)
 	dna.features["tail_lizard"] = "Smooth"
 	dna.features["pod_hair"] = "Ivy"
 
+/// Provides a dummy for unit_tests that functions like a normal human, but with a standardized appearance
+/// Copies the stock dna setup from the dummy/consistent type
+/mob/living/carbon/human/unit_test
+
+/mob/living/carbon/human/unit_test/setup_human_dna()
+	create_dna(src)
+	dna.initialize_dna(skip_index = TRUE)
```

Comment:
This needs to be a proc instead of a copy paste, I change these often enough to warrant it